From 09a3aa6e85902ef036fc61ca468b63a8c28ff9f3 Mon Sep 17 00:00:00 2001
From: Joshua <jqshuv@gmail.com>
Date: Fri, 13 May 2022 10:28:51 +0200
Subject: [PATCH] [TEMP] Change Version

---
 build.gradle | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build.gradle b/build.gradle
index 7fabe39..e194341 100644
--- a/build.gradle
+++ b/build.gradle
@@ -7,7 +7,7 @@ plugins {
 }
 
 group = 'atm.bloodworkxgaming'
-version = '2.3.1'
+version = '2.3.4'
 
 sourceCompatibility = 1.8
 
-- 
2.35.3.windows.1

